packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -154,6 +155,12 @@ mutation OpenBrowserList_SetBrowser($id: ID!) {
 }
 `
 
+gql`
```

Comment:
Unused, as the `Close` button emits an event to the parent in order to execute the mutation.